lib/linguist/languages.yml

```diff
@@ -2903,7 +2903,7 @@ OpenType Feature File:
   - ".fea"
   tm_scope: source.opentype
   ace_mode: text
-  language_id: 266
+  language_id: 374317347
```

Comment:
That's correct. This is set automagically by `script/set-language-ids` and its the result of hashing the name.